apps/dashboard/app/(app)/ratelimits/[namespaceId]/overrides/[overrideId]/settings.tsx

```diff
@@ -151,13 +176,47 @@ export const UpdateCard: React.FC<Props> = ({ overrideId, defaultValues }) => {
               )}
             />
           </CardContent>
-          <CardFooter className="flex justify-end">
+
+          <CardFooter className="flex justify-end space-x-4">
+            <Button type="button" onClick={() => setOpen(!open)} variant="alert">
+              Delete Key
+            </Button>
             <Button disabled={update.isLoading || !form.formState.isValid} type="submit">
               {update.isLoading ? <Loading /> : "Save"}
             </Button>
           </CardFooter>
         </form>
       </Form>
+      <Dialog open={open} onOpenChange={(o) => setOpen(o)}>
+        <DialogContent className="border-alert">
+          <DialogHeader>
+            <DialogTitle>Delete Override</DialogTitle>
+            <DialogDescription>
+              This override will be deleted. This action cannot be undone.
+            </DialogDescription>
+          </DialogHeader>
+
+          <Alert variant="alert">
```

Comment:
No need for the variant to be "alert", just default

Reply:
Changed